extide/gradle/nbproject/org-netbeans-modules-gradle.sig

```diff
@@ -442,6 +442,39 @@ meth public static org.netbeans.modules.gradle.api.execute.GradleCommandLine$Sta
 meth public static org.netbeans.modules.gradle.api.execute.GradleCommandLine$StackTrace[] values()
 supr java.lang.Enum<org.netbeans.modules.gradle.api.execute.GradleCommandLine$StackTrace>
 
+CLSS public final org.netbeans.modules.gradle.api.execute.GradleDistributionManager
```

Comment:
Up to @lkishalmi to be satisfied with the new API.

Reply:
Well, I am satisfied with that one.